extras/backup/substitution-rules.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+substitutionRules:
```

Comment:
Please use transformation rule instead so that we can easily add more rules to handle autopilot preflight incompatibility issue later.
The alternative transformationRules:
```
transformationRules:
- description: Recover one replica to StatefulSets
  resourceFilter:
    groupKinds:
    - resourceGroup: apps
      resourceKind: StatefulSet
    namespaces:
    - default
  fieldActions:
  - op: REPLACE
    path: "/spec/replicas"
    value: '1'
```